stan/math/fwd/fun/inv_sqrt.hpp

```diff
@@ -3,14 +3,14 @@
 
 #include <stan/math/fwd/meta.hpp>
 #include <stan/math/fwd/core.hpp>
+#include <stan/math/prim/fun/inv_sqrt.hpp>
 #include <cmath>
 
 namespace stan {
 namespace math {
 
 template <typename T>
 inline fvar<T> inv_sqrt(const fvar<T>& x) {
-  using std::sqrt;
```

Comment:
I think this file should include sqrt? or the computation should call inv_sqrt on x.val, not use sqrt?

Reply:
Included `fwd/fun/sqrt.hpp`